source/_components/toon.markdown

```diff
@@ -21,52 +21,52 @@ redirect_from:
   - /components/switch.toon/
 ---
 
-The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat and sensors for power and gas consumption. The component also auto-detects any smart plugs, solar panels and smoke detectors connected to Toon and adds sensors and switches for them.
+The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat, sensors for power and gas consumption, sensors for solar production and several binary sensors for things like boiler burner on/off, hot tap water and boiler health status.
+
+For the `toon` component to work, you'll need an active Toon subscription with Eneco and a Toon API developer account.
 
 There is currently support for the following device types within Home Assistant:
 
+- Binary Sensor
 - [Climate](#climate)
 - Sensor
-- Switch
 
-For the `toon` component to work, you'll need an active Toon subscription with Eneco. The component uses your Mijn Eneco credentials to control your thermostat through the [toonopafstand](https://toonopafstand.eneco.nl) domain.
+## {% linkable_title Setting up a developer account %}
+
+In order to be able to use this component, you'll need to sign up for a free Toon API developer account.
+
+1. Visit the [Toon API developers website](https://developer.toon.eu/), and [sign in](https://developer.toon.eu/user/login). [Create an account](https://developer.toon.eu/user/register) if you don’t have one already.
```

Comment:
```suggestion
1. Visit the [Toon API developers website](https://developer.toon.eu/), and [sign in](https://developer.toon.eu/user/login). [Create an account](https://developer.toon.eu/user/register) if you don’t have one already. These credentials do not have to be the same as your Eneco account.
```

Reply:
Not sure if that is relevant for our documentation?

Reply:
This should actually be clear enough without extra info.